modules/aws-eks-teams/main.tf

```diff
@@ -0,0 +1,235 @@
+###########
+# Namespace
```

Comment:
super nit but use existing style you can see here for sections 

https://github.com/aws-samples/aws-eks-accelerator-for-terraform/blob/main/main.tf#L32